tests/forte/advanced_pipeline_test.py

```diff
@@ -75,7 +78,7 @@ def _parse_pack(self, collection: Any) -> Iterator[MultiPack]:
 
 class DummyProcessor(PackProcessor):
     """
-    A dummy Processor to check the expected/output records from the remote
+    A dummpy Processor to check the expected/output records from the remote
```

Comment:
should be `dummy` right?

Reply:
interesting... couldn't remember I made this change.
Will revert it.